api/pkg/apps/webapp/templates/team.gohtml

```diff
@@ -43,6 +55,152 @@
         </div>
     </div>
 
+    <script>
+        let {Subject, BehaviorSubject, pipe, Observable} = rxjs;
+        let {map, debounceTime, switchMap} = rxjs.operators;
+        let {ajax} = rxjs.ajax;
+
+        // Handles the search box for the parties dropdown
+        let partySearchSubject = new Subject()
+        let partySearch$ = partySearchSubject.asObservable().pipe(
+            debounceTime(200),
+            switchMap((searchParam) => {
+                const searchParams = new URLSearchParams({
+                    "partyTypeId": "{{.StaffPartyTypeId}}",
+                    "searchParam": searchParam
+                })
+                const ajaxUrl = "/relationships/pickparty?" + searchParams.toString()
```

Comment:
I would add a specific endpoint to the webapp `/teams/addmemberpicker` to handle the specific case of selecting a user for a team (for example, filtering out members that are aldready in the team)

Reply:
I was thinking about how to implement this, in SQL I would use a join between memberships and parties. Would I need to make 2 API calls and then filter the subsequent lists manually?

Or should I create a new store method that uses the MongoDB `$lookup` as described here:
https://docs.mongodb.com/v5.0/reference/operator/aggregation/lookup/#pipe._S_lookup

Reply:
I don't think we need to filter on the database for now. Teams are likely to be quite small. In-memory filtering would be fine (retrieve team member IDs for the team, and filter them out from the party search result)